### PR TITLE
Fixing error  #24

### PR DESCRIPTION
**Pull request by paulkiren**
Sending back the Error Object, on exception.

### CODE REVIEW

**Comment by paulkiren** [comment]
@nandarustam please review it.

File: lib/fcm.js [hunk 1]
```diff
@@ -106,7 +106,11 @@ FCM.prototype.send = function(payload, CB) {
 
                     // Success, return message id (without id=), or something error happened
                     if (id) resolve(id);
-                    if (error) reject(error);
+                    if (error) {
+                      const err = new Error();
+                      err.message = error;
+                      reject(err);
```
Comment (nandarustam) [inline lib/fcm.js]: nit. to be consistent, please change to 4-space indentation

File: lib/fcm.js [hunk 2]
```diff
@@ -117,7 +121,9 @@ FCM.prototype.send = function(payload, CB) {
             });
 
             request.on('error', function(error) {
-                reject(error);
+              const err = new Error();
+              err.message = error;
+              reject(err);
```
Comment (nandarustam) [inline lib/fcm.js]: nit. same thing, 4-space indentation

File: lib/fcm.js (hunk 1 above)
Comment (paulkiren) [reply lib/fcm.js]: I will Change the Space indentation.

**Comment by paulkiren** [comment]
@nandarustam , Changes done. please do a review

**Comment by paulkiren** [comment]
@nandarustam please do a review

**Comment by nandarustam** [comment]
Patched in #48

**Pull request closed without merging**